services/IBF-pipeline/pipeline/lib/pipeline/rainfalldata.py

```diff
@@ -261,21 +259,21 @@ def findTrigger(self):
                                 how='left', op='intersects')
 
         # interpolate NaN cells
-        known = ~np.isnan(compare_gdf['forecast_time'])
+        known = ~np.isnan(compare_gdf['leadTime'])
         unknown = ~known
         z = griddata((compare_gdf['longitude'][known], compare_gdf['latitude'][known]),
-                     compare_gdf[self.TRIGGER_RP_COLNAME][known],
+                     compare_gdf['triggerLevel'][known],
                      (compare_gdf['longitude'][unknown], compare_gdf['latitude'][unknown]))
-        compare_gdf[self.TRIGGER_RP_COLNAME][unknown] = z.tolist()
+        compare_gdf['triggerLevel'][unknown] = z.tolist()
 
         compare_gdf[str(str(self.leadTimeLabel) + '_pred')] = np.where(
-            (compare_gdf['mean_by_day'] > compare_gdf[self.TRIGGER_RP_COLNAME]), 1, 0)
+            (compare_gdf['mean_by_day'] > compare_gdf['triggerLevel']), 1, 0)
```

Comment:
@jannisvisser We should move these constant strings `leadTime` and `triggerLevel` to a separate constants file. Similar to how we have enums / models in TypeScript.